ui/mirage/factories/service.js

```diff
@@ -0,0 +1,34 @@
+import { Factory, faker } from 'ember-cli-mirage';
+import { provide } from '../utils';
+
+export default Factory.extend({
+  name: id => `${faker.hacker.noun().dasherize()}-${id}-service`,
+  portLabel: () => faker.hacker.noun().dasherize(),
+  tags: () => {
+    if (!faker.random.boolean()) {
+      return provide(
+        faker.random.number({ min: 0, max: 2 }),
+        faker.hacker.noun.bind(faker.hacker.noun)
+      );
+    } else {
+      return null;
+    }
+  },
+
+  afterCreate(service) {
+    service.update({
```

Comment:
This looks like a static update. Does it have to be in the `afterCreate` hook?

Reply:
hmm, I think it’s because I was considering making it conditional, but I ended up not doing that, so I’ve removed it, thanks.